### PR TITLE
fix all jsonld examples

### PR DESCRIPTION
**Pull request by maxhbr**
The current examples in serializaiton/json_ld/examples are all except one invalid.

### CODE REVIEW

File: serialization/json_ld/examples/spdx_document3.json [hunk 1]
```diff
@@ -6,18 +6,18 @@
...
       "spdxId": "https://some.namespace#spdxdocument159",
       "creationInfo": "_:creationInfo1",
       "name": "Doc 159 - two File elements",
+      "profileConformance": ["core", "software"],
       "element": [
         "https://some.namespace#SPDXRef-Package",
-        "https://some.namespace#File1",
+        "https://some.namespace#file1",
+        "https://some.namespace#file2",
         "https://spdx.dev/elements/3F26391C#spdx-spec-v2.3",
```
Comment (maxhbr) [inline serialization/json_ld/examples/spdx_document3.json:21]: the creator `"https://spdx.dev/elements/3F26391C#spdx-spec-v2.3",` is listed in `"element"` of the SpdxDocument but is not contained in the file

**Comment by maxhbr** [comment]
the "profileConformance" value "licensing" causes the parser to fail -> got dropped.

**Comment by maxhbr** [comment]
The `SoftwareDependencyRelationship` used in `contverted_from_spdx_2.json` seems to have vanished without replacement. How would one express dynamic linking?

**Comment by maxhbr** [comment]
There are now no examples demonstrating contentIdentifier, this should be added

**Review by JPEWdev — APPROVED** [review]

**Review by goneall — APPROVED** [review]
LGTM

**Pull request merged**